src/main/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpoint.java

```diff
@@ -152,7 +152,7 @@ public QueryEndpoint(
     }
 
     @Transactional
-    @RequestMapping(value = "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases",
+    @RequestMapping(value = "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid:^[a-zA-Z0-9_.]+$}/cases",
```

Comment:
This returns a 404, which is only technically correct.
The input could be accepted while request would be dispatched to the controller, but then it could be refused with an http 400. That way, the response would be better designed against a malformed case type Id.
What is more, the above request mapping makes it possible for similar URL patterns to be configures mismatching the specific one. Then, the specific URL patterns matching different regular expressions would be confusing from a client's PoV.